### DISCUSSION TITLE
How bot get the domain list.

### DISCUSSION THREAD

**Discussion by jagadeshjai** [uBlockOrigin/uAssets#18624] [Q&A]
Hi,

Can someone help me understand how the bot gathers the domains and adds them to the list? Is it manually configured or based on any automation?

**Comment by iam-py-test** [comment]
What list?
All of the lists here are maintained by hand

**Reply by jagadeshjai** [reply]
Hi @iam-py-test 


I'm taking about these kind of commits 

https://github.com/uBlockOrigin/uAssets/commit/51a2dd1a74b932aa68f1c4a28b4c61a533d0a219

How we can identify the actual person who involved in this commit or the reason for these changes.


This may be the dumb question, but I'm not getting the exact picture of it. Kindly help to understand :smiling_face_with_tear:

**Reply by stephenhawk8054** [reply]
Those are for syncing with EasyList repo: https://github.com/easylist/easylist/

It's done by using github actions: https://github.com/uBlockOrigin/uAssets/blob/master/.github/workflows/update-easylist.yml
